x-pack/legacy/plugins/security/server/lib/saved_objects_client/secure_saved_objects_client_wrapper.test.js

```diff
@@ -995,4 +995,198 @@ describe(`spaces disabled`, () => {
       });
     });
   });
+
+  describe('#bulkUpdate', () => {
+    test(`throws decorated GeneralError when hasPrivileges rejects promise`, async () => {
+      const type = 'foo';
+      const mockErrors = createMockErrors();
+      const mockCheckPrivileges = jest.fn(async () => {
+        throw new Error('An actual error would happen here');
+      });
+      const mockCheckSavedObjectsPrivilegesWithRequest = jest.fn().mockReturnValue(mockCheckPrivileges);
+      const mockRequest = Symbol();
+      const mockAuditLogger = createMockAuditLogger();
+      const mockActions = createMockActions();
+      const client = new SecureSavedObjectsClientWrapper({
+        actions: mockActions,
+        auditLogger: mockAuditLogger,
+        baseClient: null,
+        checkSavedObjectsPrivilegesWithRequest: mockCheckSavedObjectsPrivilegesWithRequest,
+        errors: mockErrors,
+        request: mockRequest,
+        savedObjectTypes: [],
+        spaces: null,
+      });
+
+      const objects = [{
+        type
+      }];
+      await expect(
+        client.bulkUpdate(objects)
+      ).rejects.toThrowError(mockErrors.generalError);
+
+      expect(mockCheckSavedObjectsPrivilegesWithRequest).toHaveBeenCalledWith(mockRequest);
+      expect(mockCheckPrivileges).toHaveBeenCalledWith([mockActions.savedObject.get(type, 'bulk_update')], undefined);
+      expect(mockErrors.decorateGeneralError).toHaveBeenCalledTimes(1);
+      expect(mockAuditLogger.savedObjectsAuthorizationFailure).not.toHaveBeenCalled();
+      expect(mockAuditLogger.savedObjectsAuthorizationSuccess).not.toHaveBeenCalled();
+    });
+
+    test(`throws decorated GeneralError when hasPrivileges rejects promise for any one of multiple objects`, async () => {
```

Comment:
super-nit: `authorised` -> `authorized` and `unauthorised` -> `unauthorized`. I know both are technically correct, but the `z` is consistent with the rest of the suite :)

Reply:
Haha fair, my spell checker is set to English, not American. ;)